editor/editor.tsx

```diff
@@ -299,10 +300,16 @@ function Editor(props: RouteComponentProps) {
             // gosRef.current.api.subscribe('click', (type: string, eventData: CommonEventData) => {
             //     setMouseEventInfo({ type: 'click', data: eventData.data, position: eventData.genomicPosition });
             // });
+
+            // Range Select API
+            gosRef.current.api.subscribe('rangeselect', (type: string, eventData: RangeMouseEventData) => {
```

Comment:
Shouldn't need to add a type annotation for `type` or `eventData`. The types for this function should be inferred from the `"rangeselect"` event name.